vsintegration/src/FSharp.Editor/CodeFixes/CodeFixes.md

```diff
@@ -0,0 +1,50 @@
+# Code Fixes In F#
+
+Code fixes are suggestions provided by IDEs to automatically address coding issues. In Visual Studio, they fall under the category of [Quick Actions](https://learn.microsoft.com/visualstudio/ide/quick-actions) and are displayed within the light bulb icon menu.
+
+## Quick Facts
+
+1. Code fixes are triggered by one or more diagnostics (e.g., FS0039 and/or IDE0005).
```

Comment:
It might be worth describing where these come from and how to add new ones (or linking to such a description if it exists).

Reply:
Good point. I think our README on that topic is really good now so I am just linking it :)